src/meta-srv/src/test_util.rs

```diff
@@ -40,17 +42,22 @@ pub(crate) fn new_region_route(region_id: u64, peers: &[Peer], leader_node: u64)
     }
 }
 
-/// Builds and returns a [`SelectorContext`]. To access its inner state,
-/// use `memory_backend` on [`MetaPeerClientRef`].
-pub(crate) fn create_selector_context() -> SelectorContext {
+pub(crate) fn create_meta_peer_client() -> MetaPeerClientRef {
     let in_memory = Arc::new(MemoryKvBackend::new());
-    let meta_peer_client = MetaPeerClientBuilder::default()
+    MetaPeerClientBuilder::default()
         .election(None)
         .in_memory(in_memory.clone())
```

Comment:
```suggestion
        .in_memory(in_memory)
```

Reply:
resolved in [62a7c27](https://github.com/GreptimeTeam/greptimedb/pull/6055/commits/62a7c2775a6ff6745c9d5bad2b147a115367fc64)